inspector/src/modules/log_module.ts

```diff
@@ -783,18 +867,33 @@ function setEnabledFilterButtonStyle(
  * @param {string} logTxt
  * @returns {HTMLElement}
  */
-export function createLogElement(logTxt: string): HTMLElement {
+export function createLogElement(
+  logTxt: string,
+  logView: ObservableState<LogViewState>,
+  configState: ObservableState<ConfigState>,
+  ): HTMLElement {
   let namespace;
-  let formattedMsg = logTxt;
-  const indexOfNamespaceStart = logTxt.indexOf("[");
+  // let formattedMsg = logTxt;
+  const timeRegex = /^(\d+(?:.)?(?:\d+)?) (.*)$/
+  let match = logTxt.match(timeRegex)
```

Comment:
Because it may become expensive with thousands of logs, I would prefer if we called `match` after having already checked that we want to transform it as a date (it may lead to 2 `if`s though)